exe-common/src/config.rs

```diff
@@ -192,6 +192,7 @@ pub mod net {
     pub struct Config {
         pub genesis: HeaderHash,
         pub genesis_prev: HeaderHash,
+        pub epoch_stability_depth: u32,
```

Comment:
I still would like to see `usize` here, or to understand why this special `u32` type?